changes/xxxx.housekeeping

```diff
@@ -0,0 +1,10 @@
+Updated development dependency `django-debug-toolbar` to `~6.0.0`.
```

Comment:
May be undesirable, see: https://github.com/django-commons/django-debug-toolbar/issues/2168